EDEngineer/Resources/Data/entryData.json

```diff
@@ -1807,6 +1807,17 @@
     "Group": "ThargoidShip",
     "MaximumCapacity": 200
   },
+  {
+    "Name": "Massive Energy Surge Analytics",
+    "Rarity": "Rare",
+    "FormattedName": "tg_shutdowndata",
+    "Kind": "Data",
+    "OriginDetails": [
+      "Ship scanning (Thargoid interceptors)"

```

Comment:
- you don't get the data by scanning it. You simply need to get hit by a huge, greeny wave in order to achieve the data;
- it's from the Titan one (inside the heart of a maelstrom), not interceptors.

**note:**
_any kind of error from me providing information about the data might come from the lack of my knowledge about thargoids._